_layouts/i18n.html

```diff
@@ -43,12 +43,6 @@
             <a class="bold" href="#" onclick="return scrollToResults('popular-div');">Popular</a>&emsp;
             <a class="bold" href="#" onclick="return scrollToResults('recent-div');">Recent</a>
           </td>
-
```

Comment:
You need to restore the whole table here to how it was, as it was localized as well, with "Popular" and "Recent" being injected by the jekyll build before you made your first PR:

https://github.com/unfoldingWord-dev/door43.org/pull/644/files#diff-a1d1c77400162e6d7dacfe0257e22058L42